pmg/views.py

```diff
@@ -218,10 +226,24 @@ def committee_detail(committee_id):
     recent_questions = load_from_api(
         'minister-questions-combined',
         params=params)['results']
+    all_meetings = filter(lambda m: m['type'] == 'committee-meeting' and format_date(m['date']).year >= YEAR_CUTOFF,committee['events'])
```

Comment:
A more pythonic way of doing this is to use a list comprehension, which is generally preferred to using filter:

```python
all_meetings = [m for m in committee['events'] if m['type'] == 'committee-meeting' and format_date(m['date']).year >= YEAR_CUTOFF]
```